app/page.tsx

```diff
@@ -105,6 +108,16 @@ export default function Page() {
           />
         </div>
       </div>
+    
+      <div className="flex flex-col items-center justify-center p-4 ">
+        <h1 className="mb-4 text-2xl font-bold">Custom GRID Cards Type 1</h1>
+        <GridCardType1 items={GridData.full.slice(0, 3)} />
```

Comment:
you should declare a constant for this like Ex: const itemData = GridData.full.slice(0, 3); for prevent recalculate many times, can use useMemo, to optimize performance